src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -686,6 +686,7 @@ Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPrefixIncrement = 77
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveGenericsOptions(Microsoft.CodeAnalysis.SymbolDisplayGenericsOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveLocalOptions(Microsoft.CodeAnalysis.SymbolDisplayLocalOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveMiscellaneousOptions(Microsoft.CodeAnalysis.SymbolDisplayMiscellaneousOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
+Microsoft.CodeAnalysis.SymbolDisplayFormat.WithTypeQualificationStyle(Microsoft.CodeAnalysis.SymbolDisplayTypeQualificationStyle typeQualificationStyle) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
```

Comment:
does this need to be in compiler public API? is it something that will be used more than in code lens? @gafter